content-src/asrouter/templates/CFR/templates/ExtensionDoorhanger.schema.json

```diff
@@ -212,10 +218,8 @@
                 "data": {
                   "properties": {
                     "url": {
-                      "allOf": [
-                        {"$ref": "#/definitions/linkUrl"},
-                        {"description": "URL used in combination with the primary action dispatched."}
-                      ]
+                      "type": "null",
+                      "description": "URL used in combination with the primary action dispatched."
```

Comment:
You could add a note about how this can be dynamically generated from the addon.id, see (filename)